Source/WebCore/PAL/PAL.xcodeproj/project.pbxproj

```diff
@@ -369,6 +369,13 @@
 /* End PBXBuildRule section */
 
 /* Begin PBXContainerItemProxy section */
+		074AD71C2B61795800FFA67B /* PBXContainerItemProxy */ = {
```

Comment:
What's going on in this file 😅

Reply:
literally no idea whatsoever, I’ll try to put it back